fibonacci.py

```diff
@@ -0,0 +1,9 @@
+f1=1

```

Comment:
fobonacci series starts from zero number, it prints from 3 onwards. code can be optimised.